_targets.R

```diff
@@ -24,13 +27,24 @@ list(
   # static branching with tar_map()
   tar_map(
     ## task names passed into tar_map() via arg. 'values='
-    values = tibble(state_abb = states),
+    values = tibble(state_abb = states) %>%
+      mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
+
     tar_target(nwis_inventory, dplyr::filter(oldest_active_sites,
                                              state_cd == state_abb)),
 
     tar_target(nwis_data, get_site_data(nwis_inventory,
                                         state_abb,
-                                        parameter))
+                                        parameter)),
+
+    tar_target(tally, tally_site_obs(nwis_data)),
+
+    tar_target(timeseries_png, plot_site_data(out_file = state_plot_files,
```

Comment:
Small change - this should be a target of type "file" since `plot_site_data` creates a [PNG file and returns the filename as output](https://github.com/msleckman/ds-pipelines-targets-3/blob/appliers/3_visualize/src/plot_site_data.R#L11).